File: Problem 327P1/0327_Problem_1.exs
# Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
# LeetCode Benchmark: 404 ms

def maximum_count(nums) do
  max(nums |> Enum.count(&(&1 > 0)),
      nums |> Enum.count(&(&1 < 0)))
end

File: Problem 327P1/0327_Problem_1.erl
% Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
% LeetCode Benchmark: 229 ms

maximum_count(Nums) ->
    max(length(lists:filter(fun(X) -> X > 0 end, Nums)),
        length(lists:filter(fun(X) -> X < 0 end, Nums))).
